src/msgs/measurements.rs

```diff
@@ -228,18 +235,47 @@ pub enum ContentChanged {
     False,
 }
 
+#[derive(Debug)]
+pub struct ParseContentChangedError;
+
+impl TryFrom<u8> for ContentChanged {
+    type Error = ParseContentChangedError;
+    fn try_from(value: u8) -> Result<Self, Self::Error> {
+        Ok(match value {
+            0b00 => ContentChanged::NotSupported,
+            0b01 => ContentChanged::True,
+            0b10 => ContentChanged::False,
```

Comment:
The shift down is partially due to the way we construct the bytes. I agree there is some asymmetry here. I think part of the problem are the `get_bit(s)` methods that read just the specified bits as a `u8`. Since we are going to get rid of those I'll hold off on changing this for now.